molecule/plain-rhel/verify.yml

```diff
@@ -191,6 +191,30 @@
         property: log4j.rootLogger
         expected_value: "INFO, connectAppender"
 
+    - name: Get stats on installed remote plugin
+      stat:
+        path: /usr/share/java/confluentinc-kafka-connect-s3
+      register: installed_remote_plugin
+
+    - name: Assert remote plugin is installed correctly
+      assert:
+        that:
+          - installed_remote_plugin.stat.exists
+          - installed_remote_plugin.stat.gr_name == 'confluent'
+          - installed_remote_plugin.stat.pw_name == 'cp-kafka-connect'
+        quiet: true
+
+    - name: Grab installed remote plugin manifest file
+      shell: cat /usr/share/java/confluentinc-kafka-connect-s3/manifest.json
+      register: result
+    - name: Parse manifest file
+      set_fact:
+        jsondata: "{{ result.stdout | from_json }}"
+    - name: Assert installed remote plugin version
+      assert:
+        that:
+          - jsondata.version == '10.3.0'
```

Comment:
Can we make version number as a variable ?

Reply:
yes done, thx